erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -969,6 +969,19 @@ def get_materials_from_other_locations(item, warehouses, new_mr_items, company):
 			new_mr_items.append(new_dict)
 
 	if required_qty:
+		stock_uom, purchase_uom = frappe.db.get_value('Item', item['item_code'], ['stock_uom', 'purchase_uom'])
+
+		if stock_uom != purchase_uom and purchase_uom == item['uom']:
+			conversion_factor = get_uom_conversion_factor(item['item_code'], item['uom'])
+			if not (conversion_factor or frappe.flags.show_qty_in_stock_uom):
+				frappe.throw(_("UOM Conversion factor ({0} -> {1}) not found for item: {2}")
+					.format(purchase_uom, stock_uom, item['item_code']))
+
+			required_qty = required_qty / required_qty
```

Comment:
This doesn't seem correct. Meant to use conversion factor here?

Reply:
@ankush I'm fixing that line, thank you for the comment!